app.py

```diff
@@ -1,94 +1,35 @@
 import streamlit as st
 import streamlit.components.v1 as components
 
-# Define the list of words/phrases you want to cycle through
-words = [
-    "Skibidi",
-    "Toilet"
-]
 
-# HTML and CSS for the typewriter effect
-typewriter_html = f"""
-<!DOCTYPE html>
-<html>
-<head>
-    <style>
-        .typewriter {{
-            font-family: 'Courier New', Courier, monospace;
-            color: #FF6600; /* Nixie Orange */
-            overflow: hidden; /* Ensures the text is not revealed until the animation */
-            border-right: .15em solid #FF6600; /* The typewriter cursor in Nixie Orange */
-            white-space: nowrap; /* Prevents the text from wrapping */
-            margin: 0 auto; /* Centers the element */
-            letter-spacing: .15em; /* Adjusts spacing between letters */
-            font-size: 2em; /* Adjust font size as needed */
-        }}
-        /* Optional: Center the typewriter text */
-        body {{
-            display: flex;
-            justify-content: center;
-            align-items: center;
-            height: 100%;
-        }}
-    </style>
-</head>
-<body>
-
-    <div class="typewriter" id="typewriter"></div>
-
-    <script>
-        const words = {words};
-        let currentWordIndex = 0;
-        let currentCharIndex = 0;
-        let isDeleting = false;
-        const typingSpeed = 150; // milliseconds
-        const deletingSpeed = 100;
-        const pauseBetweenWords = 1500;
-
-        const typewriterElement = document.getElementById('typewriter');
-
-        function type() {{
-            const currentWord = words[currentWordIndex];
-            if (isDeleting) {{
-                typewriterElement.innerHTML = currentWord.substring(0, currentCharIndex--);
-                if (currentCharIndex < 0) {{
-                    isDeleting = false;
-                    currentWordIndex = (currentWordIndex + 1) % words.length;
-                    setTimeout(type, 500);
-                }} else {{
-                    setTimeout(type, deletingSpeed);
-                }}
-            }} else {{
-                typewriterElement.innerHTML = currentWord.substring(0, currentCharIndex++);
-                if (currentCharIndex > currentWord.length) {{
-                    isDeleting = true;
-                    setTimeout(type, pauseBetweenWords);
-                }} else {{
-                    setTimeout(type, typingSpeed);
-                }}
-            }}
-        }}
-
-        // Start the typing effect
-        type();
-    </script>
-
-</body>
-</html>
-"""
-
-# Render the typewriter HTML in the Streamlit app
-components.html(typewriter_html, height=50)
 
+#sidebar
+st.sidebar.title("Navigation")
+page = st.sidebar.selectbox(
+    "Choose a page:",
+    ("Project Description", "User Guide", "Data")
+)
```

Comment:
@dleiferives Do you think we should have a dropbox rather than standalone boxes?

Reply:
I think that we should have standalone boxes for the moment. I expect that there will be a need for a sidebar/ side panel at some point. I imagine that mixing drop downs with a side panel would be annoying as a user. @TeachMeTW lmk if you agree, iff you disagree then I think a dropbox is should be perfect.

Reply:
@dleiferives I agree it be too confusing, so please change this to standalone boxes in the side bar @RonanValadez35

Reply:
Something like:
<img width="56" alt="image" src="https://github.com/user-attachments/assets/13c78760-7173-4831-9667-8bfb80766974">